### PR TITLE
Exported `AppSettings` for CLI and other Apps to use

### LINKED ISSUE #840: Export `AppSettings` for CLI and other Apps to use
We might have missed exporting the settings folder 😋 
https://github.com/zowe/imperative/blob/master/packages/index.ts

```typescript
export * from "./security";
```

### PR DESCRIPTION
**Pull request by aadityasinha-dotcom**
Resolves #840

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## `5.7.3`
```
Comment (zFernand0) [inline CHANGELOG.md:5]: Please change this to `## Recent Changes`?

**Review by zFernand0 — CHANGES_REQUESTED** [review]

**Review by zFernand0 — APPROVED** [review]

**Comment by zFernand0** [comment]
It seems like the header is case sensitive.
https://github.com/zowe/imperative/blob/4170db85d96dda0b741e2a83e1ff93c35aa55550/.github/workflows/changelog.yml#L24

**Review by zFernand0 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Exported `App-Settings` for cli and other apps to use [#840](https://github.com/zowe/imperative/issues/840)
```
Comment (t1m0thyj) [inline CHANGELOG.md:7]: Could you please change `App-Settings` to `AppSettings` to match the name of the interface? Sorry for being picky 😢

Comment (aadityasinha-dotcom) [reply CHANGELOG.md:7]: okay

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**